.changes/nextrelease/bugfix-php-v81-compatibility.json

```diff
@@ -0,0 +1,12 @@
+[
+    {
+        "type": "bugfix",
+        "category": "",
+        "description": "This release adds #[ReturnTypeWillChange] attribute where it is needed for PHP 8.1 compatibility"
+    },
+    {
```

Comment:
While I personally think this should be valid syntax, it's not in the current state of the SDK.  Please condense the two entries into one.  Something along the lines of 

```
[
    {
        "type": "bugfix",
        "category": "",
        "description": "This release adds #[ReturnTypeWillChange] attribute where it is needed and fixes a deprecation notice when pass null to rawurlencode() for PHP 8.1 compatibility"
    }
 ]
 
```

Reply:
Sorry to be a pain about this

Reply:
hi @SamRemis , no problem :)